data-models/ambiance/illuminance/schema.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/ambiance/illuminance/schema.json",
+    "$defs": {
+        "light": {
+            "measurementType": "ambiance/illuminance/light/lux",
+            "title": "Light",
+            "unit": "lux",
+            "type": "integer",
+            "description": "The light in lux."
+        },
+        "brightness": {
+            "measurementType": "ambiance/illuminance/brightness/percent",
+            "title": "Brightness",
+            "unit": "%",
+            "type": "integer",
+            "description": "The perceived brightness in percent."
```

Comment:
We could introduce a `minimum` and `maximum` here. 

Also could use a `number` if we want to support floating point numbers.